### ISSUE TITLE
Add CI

### ISSUE THREAD

**Issue by domoritz** [type-schema/json-schema#2]
I would recommend adding support for running automatic builds on Github actions.

**Comment by nonara** [comment]
Definitely. Will be adding Travis CI, coveralls, and snyk before publishing!

**Comment by domoritz** [comment]
I'd recommend github actions over travis for new projects but it's not a big difference.

**Comment by nonara** [comment]
> I'd recommend github actions over travis for new projects but it's not a big difference.

Ok. Will have a look! Thanks.
